.idea/Doogking.iml

```diff
@@ -16,7 +16,7 @@
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="inheritedJdk" />
```

Comment:
Better to not mix this stuff in the pull request. Please re-do de commit without changes in .idea files.

Reply:
In fact, shouldn't we keep the .idea directory entirely out of the repository? I don't know if it's the case, but there may be absolute paths and system-dependent configs that could break on other systems.